.github/workflows/version-bump.yml

```
@@ -0,0 +1,150 @@
+          git config --local user.name "GitHub Action"
+
+      - name: Determine version bump type
+        id: bump-type
+        env:
+          PR_TITLE: ${{ github.event.pull_request.title }}
+          PR_NUMBER: ${{ github.event.pull_request.number }}
+        run: |
+          # Enable strict shell error handling
+          set -euo pipefail
+          IFS=$'\n\t'
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)